src/test/java/pet/AddPetTest.java

```diff
@@ -1,31 +1,44 @@
 package pet;
 
 import client.PetClient;
+import data.PetInfo;
 import dto.requests.pet.Pet;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.IOException;
 
-import static data.PetInfo.postPet;
+import static data.PetInfo.addingPet;
 
 public class AddPetTest {
 
     public static final Logger LOGGER = LoggerFactory.getLogger(AddPetTest.class);
 
-    @Test
-    public void getPetTest() throws IOException {
+    @Test(dataProvider = "positiveTests")
+    public void getPetTest(Pet simplePet) throws IOException {
         LOGGER.info("START TEST add pet to the store ");
 
-        Pet response = PetClient.postPet(postPet());
+        Pet response = PetClient.postPet(addingPet());
 
-        Assert.assertEquals(response.getName(), postPet().getName());
-        Assert.assertEquals(response.getStatus(), postPet().getStatus());
-        Assert.assertEquals(response.getCategory().getName(), postPet().getCategory().getName());
-        Assert.assertEquals(response.getCategory().getId(), postPet().getCategory().getId());
+        Assert.assertNotNull(response.getId());
+        Assert.assertEquals(response.getName(), simplePet.getName());
+        Assert.assertEquals(response.getStatus(), simplePet.getStatus());
+        Assert.assertEquals(response.getCategory().getName(), simplePet.getCategory().getName());
+        Assert.assertEquals(response.getCategory().getId(), simplePet.getCategory().getId());
+        Assert.assertEquals(response.getPhotoUrls().toString(), simplePet.getPhotoUrls().toString());
+        Assert.assertEquals(response.getTags().size(), simplePet.getTags().size());
 
         LOGGER.info("END TEST");
     }
+
+    @DataProvider
+    public Object[][] positiveTests() {
+        return new Object[][]{
+            {PetInfo.addingPet()},
```

Comment:
Are these all the positive cases for posting a pet?

Reply:
I put it it the getPet tests because it uses different classes